src/Stratis.Bitcoin.Api.Tests/Stratis.Bitcoin.Api.Tests.csproj

```diff
@@ -9,12 +9,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0-preview-20200203-01" />
```

Comment:
There are a few more places with preview lets fix them all.

Reply:
Fixed them all here, sorry about that.